data/delegation/store.go

```diff
@@ -0,0 +1,391 @@
+package delegation
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/Oneledger/protocol/data/balance"
+	"github.com/Oneledger/protocol/data/keys"
+	"github.com/Oneledger/protocol/serialize"
+	"github.com/Oneledger/protocol/storage"
+)
+
+type DelegationStore struct {
+	state  *storage.State
+	szlr   serialize.Serializer
+	prefix []byte
+	mux    sync.Mutex
+}
+
+type MatureBlock struct {
+	Height int64
```

Comment:
We keep this Height field but seems it's used no where.

Reply:
https://github.com/Oneledger/protocol/blob/4cb6108ed75db4426c4407a40f931ad19b7ebeca/data/delegation/store.go#L156
https://github.com/Oneledger/protocol/blob/4cb6108ed75db4426c4407a40f931ad19b7ebeca/identity/validator_set.go#L370